server.go

```diff
@@ -372,7 +432,17 @@ func (s *server) handleOpenChanReq(req *openChanReq) {
 	go func() {
 		// TODO(roasbeef): server semaphore to restrict num goroutines
 		fundingID, err := s.fundingMgr.initFundingWorkflow(targetPeer, req)
-
+		if err == nil {
+			// ROUTING ADDED
+			fmt.Println("****** targetPeer", targetPeer)
```

Comment:
Logging messages should use the sub-system's `log` object. In this case, `fndgLog` should be used in at an appropriate logging level (`info`, `debug`, or `trace`).